frontend/packages/dev-console/src/components/import/image-search/ImageStream.tsx

```diff
@@ -0,0 +1,192 @@
+    if (imageNamespace.value !== selectedProject) {
+      setselectedProject(imageNamespace.value);
+      setFieldValue('imageStream.image', '');
+      setFieldValue('imageStream.tag', '');
+      setFieldValue('isi', initialValues.isi);
+    }
+    if (imageName.value !== selectedName) {
+      setselectedName(imageName.value);
+      setFieldValue('imageStream.tag', '');
+    }
+    if (imageTag.value !== selectedTag) {
+      setselectedTag(imageTag.value);
+      imageTag.value !== '' && searchImageTag();
+    }
+  }, [
+    imageNamespace,
+    imageName,
+    imageTag,
+    selectedProject,
+    selectedName,
+    selectedTag,
+    setFieldValue,
+    initialValues.isi,
+    searchImageTag,
+  ]);
+
+  return (
+    <>
+      <div className="row">
+        <div className="col-lg-4 col-md-4 col-sm-4 col-xs-12">
+          <DropdownField
+            name="imageStream.namespace"
+            label="Projects"
+            items={projectList}
+            title="Project"
+            fullWidth
+            required
+          />
+        </div>
+        <div className="col-lg-4 col-md-4 col-sm-4 col-xs-12">
+          <DropdownField
+            name="imageStream.image"
+            label="ImageStreams"
+            items={getImageStreamByNamespace(values.imageStream.namespace)}
+            title="ImageStreams"
+            fullWidth
+            disabled={values.imageStream.namespace === ''}
+            required
+          />
+          <div className="odc-imagestream-separator">/</div>
+        </div>
+        <div className="col-lg-4 col-md-4 col-sm-4 col-xs-12">
+          <DropdownField
+            name="imageStream.tag"
+            label="Tag"
+            items={getImageStreamTags(values.imageStream.image)}
+            title="Tag"
+            disabled={values.imageStream.image === ''}
+            fullWidth
+            required
+          />
+          <div className="odc-imagestream-separator">:</div>
+        </div>
+      </div>
+      {values.registry === registryType.Internal &&
+        imageNamespace.value &&
+        imageNamespace.value !== builderImagesNamespace.Openshift &&
+        values.project.name !== imageNamespace.value && (
+          <div className="row">
+            <div className="col-lg-12">
+              <Alert
+                variant="warning"
+                title="add cluster policy to use internal imagestream"
+                isInline
+              >
+                Service account <strong>default</strong> will need image pull authority to deploy
+                images from <strong>{imageNamespace.value}</strong>. You can grant authority with
+                the command:
+                <p>
+                  <code>
+                    oc policy add-role-to-user system:image-puller system:serviceaccount:
+                    {values.project.name}:default -n {imageNamespace.value}
+                  </code>
+                </p>
+              </Alert>
+            </div>
+          </div>
```

Comment:
It would be better if we can make it a sperate component. will look clean. WDYT?